src/gui/log/loglistview.cpp

```diff
@@ -88,24 +88,24 @@ namespace
 
             painter->setPen(originalPen);
             const QFontMetrics fontMetrics = painter->fontMetrics(); // option.fontMetrics adds extra padding to QFontMetrics::width
-            const int separatorLeft = horizontalAdvance(fontMetrics, time);
-            style->drawItemText(painter, textRect.adjusted(separatorLeft, 0, 0, 0), option.displayAlignment, option.palette
+            const int separatorCoordinateX = horizontalAdvance(fontMetrics, time);
+            style->drawItemText(painter, textRect.adjusted(separatorCoordinateX, 0, 0, 0), option.displayAlignment, option.palette
                                 , (option.state & QStyle::State_Enabled), SEPARATOR);
 
             coloredPen.setColor(index.data(BaseLogModel::ForegroundRole).value<QColor>());
             painter->setPen(coloredPen);
-            const int messageLeft = (separatorLeft + horizontalAdvance(fontMetrics, SEPARATOR));
-            style->drawItemText(painter, textRect.adjusted(messageLeft, 0, 0, 0), option.displayAlignment, option.palette
+            const int messageCoordinateX = separatorCoordinateX + horizontalAdvance(fontMetrics, SEPARATOR);
+            style->drawItemText(painter, textRect.adjusted(messageCoordinateX, 0, 0, 0), option.displayAlignment, option.palette
                                 , (option.state & QStyle::State_Enabled), index.data(BaseLogModel::MessageRole).toString());
             painter->restore();
         }
 
         QSize sizeHint(const QStyleOptionViewItem &option, const QModelIndex &index) const override
         {
-            const QSize fontSize = option.fontMetrics.size(0, logText(index)) + QSize(4, 4);
+            static const QSize minimumFontPadding(4, 4);
```

Comment:
why the `static`? It can be removed IMO.

Reply:
Or declare it in anonymous namespace above...